plugins/tech-insights/package.json

```diff
@@ -38,6 +38,8 @@
     "@material-ui/core": "^4.12.2",
     "@material-ui/icons": "^4.9.1",
     "@material-ui/lab": "4.0.0-alpha.57",
+    "@types/luxon": "^3.0.1",
+    "luxon": "^3.0.3",
```

Comment:
should set both of these to `^3.0.0` which is what the rest of the repo uses; the extra resolutions in yarn.lock should not be necessary

Reply:
I will be removing the Luxon dependency because we will directly return a JsonValue